frontend/src/components/home/WorkoutCard.tsx

```diff
@@ -1,15 +1,16 @@
 import {TypeWorkoutCard} from "../../interfaces/types.ts";
 import "./WorkoutCard.css";
+import {useNavigate} from "react-router-dom";
 
 function WorkoutCard (props: TypeWorkoutCard) {
+    const navigate = useNavigate();
 
     return (
-        <section className={"workout-card"}>
+        <section onClick={() => navigate("/workout/" + props.workout.id)} className={"workout-card"}>
```

Comment:
Semantically speaking, it is not good practice, to have the `onClick` event on the section tag here. No screen reader will see and can access this link.

Highly recommend using the `Link` component here to be safe, semantically.

Reply:
Checken wir wenn es soweit ist. Ist aktuell Outdated.

Reply:
> Checken wir wenn es soweit ist. Ist aktuell Outdated.

Wenn es outdated ist, sollte es nicht teil des pull request in den `main` sein, was ja euer Production Code ist. Der code, der ausgeliefert wird. An den Kunden oder Endnutzer.